cwms-radar-model/README.md

```diff
@@ -0,0 +1,12 @@
+# Generating Updated Swagger YAML for CDAC
+
+## Steps
```

Comment:
These are documentation nitpicks:

would prefer to use markdown linking and code references. For example
- `[Swagger Editor](https://editor.swagger.io/)`
- link to the local yaml file (relative to this readme)
- gradle tasks would be `./gradlew generateSwaggerCodeCwmsRadar` (this renders nicely with a clickable button in IntelliJ)

Reply:
Updated README as suggested